profiles/deviceSettingsExtendedEnumsNotSupported.yaml

```diff
@@ -0,0 +1,19 @@
+---
+dsAudio:
```

Comment:
district .yaml files required for each module.

- dsAudio.yml
- dsHdmiIn.yml

etc.

Reply:
can you upgrade main, so that all other tests are disabled when you provide a dsAudio profile, only audioFunctions should be enabled.

Same for all other modules.

Reply:
Ok let’s raise an issue to fix this the test will break if we merge though